WordPress/src/main/java/org/wordpress/android/ui/bloganuary/learnmore/BloganuaryNudgeLearnMoreOverlayUiState.kt

```diff
@@ -0,0 +1,24 @@
+package org.wordpress.android.ui.bloganuary.learnmore
+
+import androidx.annotation.StringRes
+import org.wordpress.android.R
+import org.wordpress.android.ui.utils.UiString
+
+data class BloganuaryNudgeLearnMoreOverlayUiState(
+    val noteText: UiString,
+    val action: BloganuaryNudgeLearnMoreOverlayAction,
+)
+
+enum class BloganuaryNudgeLearnMoreOverlayAction(
+    @StringRes val textRes: Int,
+    val analyticsLabel: String,
+) {
+    DISMISS(
+        textRes = R.string.bloganuary_dashboard_nudge_overlay_action_dismiss,
+        analyticsLabel = "dismiss",
```

Comment:
Would it make sense to have a unit test that fails if someone changes the value of this literal? I'm thinking of a scenario where someone changes it by accident and we start sending a different analytics event.

Reply:
This PRs doesn't add the analytics events, but I can double-check the test code for the analytics PR (https://github.com/wordpress-mobile/WordPress-Android/pull/19706) and use a hardcoded string in the test to check against instead of using the value from the enum.